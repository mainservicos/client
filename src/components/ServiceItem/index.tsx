import * as S from './styles'

const ServiceItem = () => (
  <S.Wrapper>
    <h1>ServiceItem</h1>
  </S.Wrapper>
)

export default ServiceItem
