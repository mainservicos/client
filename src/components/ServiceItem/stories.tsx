import { Story, Meta } from '@storybook/react/types-6-0'
import ServiceItem from '.'

export default {
  title: 'ServiceItem',
  component: ServiceItem
} as Meta

export const Default: Story = () => <ServiceItem />
