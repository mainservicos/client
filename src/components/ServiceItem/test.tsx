import { render, screen } from '@testing-library/react'

import ServiceItem from '.'

describe('<ServiceItem />', () => {
  it('should render the heading', () => {
    const { container } = render(<ServiceItem />)

    expect(
      screen.getByRole('heading', { name: /ServiceItem/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
