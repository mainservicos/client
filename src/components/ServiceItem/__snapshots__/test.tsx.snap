// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServiceItem /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    ServiceItem
  </h1>
</main>
`
